mteb/tasks/Audio/AudioClassification/eng/GTZANGenre.py

```diff
@@ -0,0 +1,49 @@
+from __future__ import annotations
+
+from mteb.abstasks.Audio.AbsTaskAudioClassification import (
+    AbsTaskAudioClassification,
+)
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+
+class GTZANGenre(AbsTaskAudioClassification):
+    metadata = TaskMetadata(
+        name="GTZANGenre",
+        description="Music Genre Classification (10 classes)",
+        reference="https://huggingface.co/datasets/sanchit-gandhi/gtzan",
+        dataset={
+            "path": "sanchit-gandhi/gtzan",
```

Comment:
> It is a collection of 10 different musical genres with 100 audio files each, all having a length of 30 seconds. 

This HF dataset only has 999 samples. It seems like it's missing a sample in jazz, and this dataset author missed the note in the README from GH/Kaggle:

> Note: when downloading from kaggle, 'jazz.00054.wav' is an invalid file. You must delete it from the data you download and manually replace it with the valid version found here: https://www.kaggle.com/andradaolteanu/gtzan-dataset-music-genre-classification/discussion/158649.

I'd suggest recreating the correct dataset.